Dockerfile

```diff
@@ -85,3 +85,9 @@ COPY --from=trivy-scanner-build /workspace/out/trivy-scanner /
 USER 65532:65532
 WORKDIR /var/lib/trivy
 ENTRYPOINT ["/trivy-scanner"]
+
+FROM busybox AS empty
```

Comment:
can we use one of the distroless images here instead of busybox?

Reply:
I get this error with distroless "/bin/sh": stat /bin/sh: no such file or directory, is there a specific version of it we should use?

Reply:
ah i see, perhaps `gcr.io/distroless/static:debug-nonroot`